File: 251110/알파벳에 따른 평가/evaluation-by-alphabet.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim();

let result = null;
if (input === 'S') {
    result = 'Superior';
} else if (input === 'A') {
    result = 'Excellent';
} else if (input === 'B') {
    result = 'Good';
} else if (input === 'C') {
    result = 'Usually';
} else if (input === 'D') {
    result = 'Effort';
} else {
    result = 'Failure';
}

console.log(result);

File: 260311/알파벳에 따른 평가/evaluation-by-alphabet.py
import sys

input = sys.stdin.readline

alphabet = input().rstrip()
result = None

if alphabet == 'S':
    result = 'Superior'
elif alphabet == 'A':
    result = 'Excellent'
elif alphabet == 'B':
    result = 'Good'
elif alphabet == 'C':
    result = 'Usually'
elif alphabet == 'D':
    result = 'Effort'
else:
    result = 'Failure'

print(result)